data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java

```diff
@@ -11,49 +11,40 @@
 
 package com.amazon.dataprepper.plugins.source.loghttp;
 
-import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 import io.micrometer.core.instrument.util.StringUtils;
 
 import java.nio.file.Files;
 import java.nio.file.Path;
 
 public class HTTPSourceConfig {
-    static final String PORT = "port";
     static final String REQUEST_TIMEOUT = "request_timeout";
     static final String THREAD_COUNT = "thread_count";
-    static final String MAX_CONNECTION_COUNT = "max_connection_count";
```

Comment:
Why can we remove this?

Reply:
This string is no longer being used because the tests use a POJO. I pushed a change which added the string literal into the `@JsonProperty`. For those properties which are being used elsewhere, I kept those as constants.